src/components/Button.scss

```diff
@@ -1,20 +1,73 @@
 // TODO Move to src/styles/colors.scss
-$text: #fff;
+
+$white: #ffffff;
+$blue: #09719B;
 $normal: #09628d;
 $highlight: #09769f;
+$gradient: linear-gradient(180deg, #09719B 0%, #09719B 2.36%, #424A52 100%);
+
+.general_button {
+  width: 30%;
+  font-family: Montserrat;
```

Comment:
@banciur  made an extend for settings fonts, it's best to use these. You can find them in `src/styles/fonts`.